ui-tests/src/test/java/org/openmrs/reference/LoginTest.java

```diff
@@ -33,46 +36,45 @@ public void verifyModulesAvailableOnHomePage() throws Exception {
         assertTrue(homePage.isDataManagementAppPresent());
         assertTrue(homePage.isConfigureMetadataAppPresent());
         assertTrue(homePage.isSystemAdministrationAppPresent());
-    }
-
+    } 
+    
     @Test
-    public void verifyClerkModulesAvailableOnHomePage() throws Exception {
+    @Category(BuildTests.class)
+    public void verifyClerkModulesAvailableOnHomePage() {
         goToLoginPage().loginAsClerk();
-        homePage = new HomePage(page);
-        assertPage(homePage);
+        initiateHomePage();
         assertTrue(homePage.isActiveVisitsAppPresent());
         assertTrue(homePage.isAppointmentSchedulingAppPresent());
         assertTrue(homePage.isRegisterPatientCustomizedForRefAppPresent());
-    }
-
+    } 
```

Comment:
![screenorder](https://user-images.githubusercontent.com/48015736/123166004-945c2780-d47d-11eb-984d-4bbccfb45e12.png)
this is how it appears in eclipse, you can see it will leave on space in between the other ending line of the method